frontend/src/metabase-lib/v1/expressions/config.ts

```diff
@@ -66,7 +66,7 @@ export const MBQL_CLAUSES: MBQLClauseMap = {
   },
   offset: {
     displayName: `Offset`,
-    type: "aggregation",
+    type: "window",
```

Comment:
I decided to introduce a new return type because the `Offset` function can be used both as an `"aggregation"` and as an `"expression"` (depending on the argument it's being called with).

Ideally the `type` should be computed dynamically based on the argument the function is being called with but I decided to go with this simpler solution that fits the current framework better.